interface/patient_file/report/custom_report.php

```diff
@@ -1085,5 +789,17 @@ function next_prev(action){
 else {
 ?>
 </body>
+<?php if (!$printable) { ?>
```

Comment:
Looks like mostly moving code around?  I doubt the performance benefit will be measurable, but the modularization aspect is nice.  Towards that end it would also be cool to look into converting the php to an object model.